lib/myob/api/models/base.rb

```diff
@@ -137,7 +137,17 @@ def query_string(params)
         def build_filter(value)
           return value unless value.is_a?(Hash)
 
-          value.map { |key, value| "#{key} eq '#{value.to_s.gsub("'", "''")}'" }.join(' and ')
+          value.map do |key, nested_value|
+            if nested_value.is_a?(Array)
+              "(#{key} eq '#{nested_value.map { |i| sanitize_value(i) }.join("' or #{key} eq '")}')"
```

Comment:
Is the syntax correct? I just can't figure it out just by looking at it. Too many quotes :P

Reply:
Haha yeah, the gem doesn't have a test suite, but I've tried locally with multiple values in that array and it's working :)